wiki/Gameplay/Game_modifier/de.md

```diff
@@ -0,0 +1,89 @@
+- [Half Time (HT)](/wiki/Game_modifier/Half_Time) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+
+### Schwieriger
+
+- [Hard Rock (HR)](/wiki/Game_modifier/Hard_Rock) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- [Sudden Death (SD)](/wiki/Game_modifier/Sudden_Death) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+  - [Perfect (PF)](/wiki/Game_modifier/Perfect) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- [Double Time (DT)](/wiki/Game_modifier/Double_Time) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+  - [Nightcore (NC)](/wiki/Game_modifier/Nightcore) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- [Hidden (HD)](/wiki/Game_modifier/Hidden) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+  - [Fade In (FI)](/wiki/Game_modifier/Fade_In) ![][osu!mania]
+- [Flashlight (FL)](/wiki/Game_modifier/Flashlight) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+
+### Sonstige
+
+- [Relax (RL)](/wiki/Game_modifier/Relax) ![][osu!] ![][osu!taiko] ![][osu!catch]
+- [Autopilot (AP)](/wiki/Game_modifier/Autopilot) ![][osu!]
+- [Spun Out (SO)](/wiki/Game_modifier/Spun_Out) ![][osu!]
+- [1K, 2K, 3K, 4K, 5K, 6K, 7K, 8K, 9K (xK)](/wiki/Game_modifier/xK) ![][osu!mania]
+- [Co-op (CP)](/wiki/Game_modifier/Co-op) ![][osu!mania]
+- [Mirror (MR)](/wiki/Game_modifier/Mirror) ![][osu!mania]
+- [Random (RD)](/wiki/Game_modifier/Random) ![][osu!mania]
+- [Auto (AT)](/wiki/Game_modifier/Auto) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+  - [Cinema (CM)](/wiki/Game_modifier/Cinema) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- [ScoreV2 (SV2)](/wiki/Game_modifier/ScoreV2) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- [Target Practice (TP)](/wiki/Game_modifier/Target_Practice) ![][osu!] **nur Cutting Edge (Experimentell)**
+
+### Andere
+
+*Tipp: Diese Mods wurden vor einiger Zeit verwendet, aber sind jetzt nicht mehr verfügbar.*
+
+- [10K](/wiki/Game_modifier/10K) ![][osu!mania]
+- [Fade Out](/wiki/Game_modifier/Fade_Out) ![][osu!mania]
+- [No Video](/wiki/Game_modifier/No_Video) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+
+### Verwandte Begriffe
+
+#### NoMod
+
+In [Turnierspielen](/wiki/Tournaments) bezieht sich **NoMod** (***NM***) darauf, keine Mods zu nutzen. Bei vielen Turnieren müssen einige Mods standardmäßig als Teil der Regeln oder des Formats verwendet werden, wie z. B. [No Fail](/wiki/Game_modifier/No_Fail) oder [ScoreV2](/wiki/Game_modifier/ScoreV2), die eine Ausnahme zu diesem Konzept darstellen.
+
+#### FreeMod
+
+In [Turnierspielen](/wiki/Tournaments) bezieht sich **FreeMod** (***FM***) darauf, dass die Mods oder Modkombinationen freigewählt werden können. Einige Turniere sehen ebenfalls Regeln vor, die zusätzliche Kriterien bestimmen, wie z. B. welche Mods erlaubt sind und in welchen Kombinationen oder ob es erlaubt ist, keine Mods zu haben, wenn FreeMod festgelegt ist.
```

Comment:
```suggestion
In [Turnierspielen](/wiki/Tournaments) bezieht sich **FreeMod** (***FM***) darauf, dass die Mods oder Modkombinationen freigewählt werden können. Einige Turniere sehen ebenfalls Regeln vor, die zusätzliche Kriterien bestimmen, wie z. B. welche Mods erlaubt sind und in welchen Kombinationen oder ob es erlaubt ist, keine Mods zu nutzen, wenn FreeMod festgelegt ist.
```
I know the english article says `having` but I feel like this makes more sense. :thinking: